### DISCUSSION TITLE
Help with Home Assistant setup

### DISCUSSION THREAD

**Discussion by chrismaki** [mrlt8/docker-wyze-bridge#1061] [Q&A]
Hi everyone,

I've gone through a bunch of documentation, but I still can't this add-on to function correctly. I had to change the default RTSP stream port to 8556 due to conflicts with _WebRTC Camera_ HACS Integration but otherwise a pretty basic setup so far.

```
WYZE_EMAIL: user@host.com
WYZE_PASSWORD: password
NET_MODE: ANY
SNAPSHOT: Disable
MQTT_DTOPIC: homeassistant
ENABLE_AUDIO: false
MOTION_API: false
ON_DEMAND: true
SUBSTREAM: false
CAM_OPTIONS: []
API_ID: IDhere
API_KEY: KEYhere
```

I have 5 cameras: v2 x3, v3 x1, OG x1 (The V3 is still running the RTSP firmware from Waze)

When I first entered the API information and (re)started the add-on, I was happy to see thumbnails load for all of the cameras, but that was the most success I've had. One of the v2's has updated as of 13 minutes ago, the v3 still has the thumbnail the app had at the time of setup, but that thumbnail predates my setup time. and the other three all snapshots from roughly the time I initially setup the add-on. I've clicked all three options under the Restart button, I've tried the individual controls within the cameras (enable/disable/start/stop). For the most part the log is full of 304 messages, other than the one camera that updated 13 minutes ago. Just now I tried a couple of commands on "Attic Cam 1":

```
[WyzeBridge] [CONTROL] SET attic-cam-1 state=enable
[WyzeBridge] [CONTROL] SET attic-cam-1 state=start
[WyzeBridge] 🎉 Connecting to WyzeCam V2 - Attic Cam 1 on 10.0.0.242
[WyzeBridge] 172.30.32.2 - - [08/Dec/2023 07:21:27] "GET /api/attic-cam-1/state/start HTTP/1.1" 200 -
[attic-cam-1] 📡 Getting 180kb/s HD stream (H264/15fps) via LAN mode (WiFi: 94%) FW: 4.9.9.2847 🔒 (DTLS) (2/3)
[WyzeBridge] ✅ '/attic-cam-1 stream is UP! (3/3)
[attic-cam-1] Stream stopped
[WyzeBridge] ❌ '/attic-cam-1' stream is down
[WyzeBridge] ❌ '/attic-cam-1' stream is down
```

And while there was a success message above, the thumbnail never updated. I feel like I've likely missed something obvious in the initial setup. Obviously I have to get the built-in dashboard working before I can think about using the streams in WebRTC so if someone can point me in the right direction I would appreciate it.

Update: I just noticed when I click the pop-up in the preview thumbnail for RTSP stream, it tries to open the stream on 8554 and not 8556 even though I have the latter set in the configuration. If I try to connect to the RTSP URL using 8556 in VLC, it still does not open. Also, when I click "RTSP Snapshot" under snapshots, it loads a new one in a new tab and that updated snapshot is subsequently used in the frontend.

Update 2: If I connect to http://<HA IP>:5000/ and not via homeassistant, I can see the frontend refreshing. This got me thinking, I typically connect to my HA instance using https://hostnmae.duckdns.org/, so instead of that I went to https://<HA IP>:8123 and now the dashboard seems to work somewhat, still trying things out. The thumbnails don't auto-update though and I have to hit the green circle.

**Comment by g13092** [comment]
This doesn't explain why you would have gotten a thumbnail here and there, but I don't have SNAPSHOT: Disable in my environment variable list.  I can't tell what the default is, but suspect it is SNAPSHOT: RTSP

https://github.com/mrlt8/docker-wyze-bridge/wiki/Advanced-Option#snapshotstill-images
